### ISSUE TITLE
How do I increase the image size?

### ISSUE THREAD

**Issue by balzathor** [PaoloRotolo/AppIntro#30]
I would love to have the image size increased.
Even xxxHDPI looks too small on my tablet.

**Comment by PaoloRotolo** [comment]
Hi again @balzathor :)
I think I didn't understand which image are you referring to. Can you send a screen?

**Comment by balzathor** [comment]
Well I had many problems using your library. It is good but doesn't quite work well with what I have made. I've just reverted back to the version before that because everytime I switch activity it would show me that screen, which basically no one would've loved to see after once it was done with. Like basically the slides popped up everytime I came out of my application settings. Anyways, the image I was referring to was the slide.

I've currently moved a step back, bu I'll keep a watch on this stuff and include it back when I have some quality of free time.

Good luck with it :)
